src/es5processor.ts

```diff
@@ -260,6 +269,10 @@ class ES5Processor extends Rewriter {
       const call = decl.initializer as ts.CallExpression;
       const require = this.extractRequire(call);
       if (!require) return false;
+      // In JS transpilation, the only require('...') that needs to be
+      // rewritten is require('tslib'), which is emitted by TSC when loading
+      // ES6 polyfills.
```

Comment:
Does this mean you want to feed plain require() into Closure Compiler?  Does that work?  I feel like I'm missing some context on the motivation for this change.